compiler/prelude/THNames.hs

```diff
@@ -1041,6 +1042,10 @@ interruptibleIdKey = mkPreludeMiscIdUnique 442
 funDepIdKey :: Unique
 funDepIdKey = mkPreludeMiscIdUnique 445
 
+-- mulArrow. TODO move before merging
```

Comment:
Ah, I hadn't noticed that the first time around. But I guess you want to move this about.